plugins/arElasticSearchPlugin/lib/model/arElasticSearchInformationObjectPdo.class.php

```diff
@@ -1063,6 +1063,50 @@ private function getActRights()
     return $actRights;
   }
 
+  /**
+   * Get an array outlining which top level description the current description is part of.
+   *
+   * @return array  An array containing the id and various i18n titles of the top level description.
+   *                If this description is already a TLD or another issue occurs, return null.
+   */
+  public function getPartOf()
+  {
+    $collectionRootId = $this->getCollectionRootId();
+
+    if ($collectionRootId && $collectionRootId != $this->id)
```

Comment:
You could reverse the condition and return null directly in here, to avoid having all the code bellow indented to just return null after the if.